public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -147,6 +130,28 @@ export class UnConnectedExploreToolbar extends PureComponent<Props, {}> {
     });
   };
 
+  stopLive = () => {
+    const { exploreId } = this.props;
+    // TODO referencing this from perspective of refresh picker when there is designated button for it now is not
+    //  great. Needs another refactor.
+    this.props.changeRefreshIntervalAction({ exploreId, refreshInterval: (RefreshPicker as any).offOption.value });
```

Comment:
@dprokop Got myself into a pickle here. I wanted to reference these options as statics on `RefreshPicker` for easier import. That works fine until I wrapped it into `withTheme`. HoistNonReactStatics works but the types are lost and typescript does not know that it still has those statics. Do you know if there is a solution for that?

Reply:
sorry, don't have ready solution for this:(

Reply:
Fixed it some extent in withTheme adding another generic param to specify the resulting statics. Not great means you need to manually define your statics again:
```typescript
export const RefreshPicker = withTheme<
  Props,
  {
    offOption: typeof RefreshPickerBase.offOption;
    liveOption: typeof RefreshPickerBase.liveOption;
  }
>(RefreshPickerBase);
```
But I assume it is fine for the rare occasions if we do not have better solution.